wpcli-dev/provision.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Provision WP-CLI develop and do an alias to the original wpcli
+
+cd "/srv/www/"
+
+noroot() {
+  sudo -EH -u "vagrant" "$@";
+}
+
+DB_NAME='wp_cli_test'
+
+echo "Downloading a developer version of WP-CLI"
+
+# Make a database, if we don't already have one
+echo -e "\nCreating database '${DB_NAME}' (if it's not already there)"
+mysql -u root --password=root -e "CREATE DATABASE IF NOT EXISTS ${DB_NAME}"
+mysql -u root --password=root -e "GRANT ALL PRIVILEGES ON ${DB_NAME}.* TO wp_cli_test@localhost IDENTIFIED BY 'password1';"
+echo -e " DB operations done.\n"
```

Comment:
Do we need a database for WP CLI development?

Reply:
Looking how the wp-cli dev environment is configured yes, I didn't investigate too much but as default values seems that in this works and we can avoid other changes.